app/assets/stylesheets/view_specific/shared/email_status.scss

```diff
@@ -1,4 +1,4 @@
-.email-status {
+.view-shared-email_status {
```

Comment:
Changing the `view-` prefix to `v-` sounds good, but I want to keep the path in there to avoid naming conflicts (e.g. `v-index`).